### PR TITLE
Feat: tp KIND

### PR DESCRIPTION
**Pull request by rguichard**

### CODE REVIEW

File: tp/containers/k8s-kubeadm-intro.fr.md [hunk 1]
```diff
@@ -119,18 +122,18 @@ EOF
 
 Sur les machines, 2 interfaces réseaux sont présents :
 
-- eth0: nécessaire à Vagrant
-- eth1: le réseau privé que nous allons utiliser pour kubeadm
+- enp0s3: nécessaire à Vagrant
```
Comment (ArchiFleKs) [inline tp/containers/k8s-kubeadm-intro.fr.md:125]: Is this always the case ? does it depends on the vagrantbox used or the vagrant version ?

Comment (rguichard) [reply tp/containers/k8s-kubeadm-intro.fr.md:125]: I'm not sure about the consistence. But for the past few months, I had enp0s3/8 each time. Anyway, the TP specifies that the name can change et such changes should be taken into consideration. At least with enp0s3/8, we have a common default choice.

File: tp/containers/k8s-kind-install.fr.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Travaux Pratiques: Kubernetes - KIND
+
+## Introduction
+
+Dans ce TP, nous allons déployer un cluster avec KIND (Kubernetes in Docker).
```
Comment (tbobm) [inline tp/containers/k8s-kind-install.fr.md:5]: Maybe a link to the kind website (https://kind.sigs.k8s.io/) could fit in the introduction ?

File: tp/containers/k8s-kind-install.fr.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Une installation pour Linux ainsi qu'une installation est disponible.
```
Comment (tbobm) [inline tp/containers/k8s-kind-install.fr.md:6]: It looks like "pour Windows" is missing here

**Review by ArchiFleKs — APPROVED** [review]

File: tp/containers/k8s-kubeadm-intro.fr.md (hunk 1 above)
Comment (tbobm) [reply tp/containers/k8s-kubeadm-intro.fr.md:125]: AFAIK it depends on the vagrantbox, starting from Ubuntu 16 the default interfaces are usually `enp0s3` and `enp0s8`
I've tested on Ubuntu 14 (`eth0` and `eth1`), Ubuntu 16+ (`enp0s3` and `enp0s8`)

(source: [systemd v197's predictable interface names](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/), Ubuntu versions prior to 14.10+ seems to be using Upstart)

**Pull request merged**